docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -3042,7 +3042,14 @@ Message bundle files must be encoded in _UTF-8_.
 The file name consists of the relevant bundle name (e.g. `msg`) and underscore followed by a language tag (IETF; e.g. `en-US`).
 The language tag may be omitted, in which case the language tag of the default bundle locale is used.
 For example, if bundle `msg` has default locale `en`, then `msg.properties` is going to be treated as `msg_en.properties`.
-If both `msg.properties` and `msg_en.properties` are detected, an exception is thrown and build fails.
+
+If there are multiple files for a specific locale then Qute attempts to resolve the ambiguity.
+Localized files from the application root have higher priority and take precedence over localized files from dependencies.
+If multiple files of the same priority exist, then the build fails.
+For example, if the default bundle locale is `en` and the files `msg.properties` and `msg_en.properties` are found in the application root, then an exception is thrown and the build fails. 
+Or another example - if there are two dependencies and both contain the `msg_en.properties` file, then the build fails again.
```

Comment:
But it is quite natural that there will be 2 dependencies with `msg_en.properties`, each with localized messages for their own messages. They do not need to overlap, unless they define the same localizations. Wouldn't it be smarter to merge them and only fail build if there are any conflicts in defined messages (like they define the same localization and with different value)?

Reply:
Ok, let's go with your version and wait for possible user change request.